pkg/lib/infra/sms/twilio/model.go

```diff
@@ -36,3 +36,16 @@ func ParseSendResponse(jsonData []byte) (*SendResponse, error) {
 	}
 	return &response, nil
 }
+
+type ErrorResponse struct {
+	Code int `json:"code,omitempty"`
```

Comment:
Other than `code`, does it have any other documented fields? Any reference you can find on their official documentation?

Reply:
I cannot find the reference, but in my testing it returns 

```
{
  "code": 21211,
  "message": "Invalid 'To' Phone Number: xxx",
  "more_info": "https://www.twilio.com/docs/errors/21211",
  "status": 400
}
```

Reply:
I found it here, you need to choose "json" to view the error in json.
https://www.twilio.com/docs/usage/twilios-response

Let me add a comment to reference this url.

Reply:
It seems that all fields are optional. If `code` is not present, treat it as unknown error.

Reply:
Yes, this is handled in parseAndHandleErrorResponse, if it is something we do not understand (Including response without a code, or even a html), we simply treat as unknown error.